Production/crab/configs/Run3L2/EphemeralHLTPhysics.txt

```diff
@@ -0,0 +1,11 @@
+lumiMask=lumisection_EphemeralHLTPhysics_json.txt
```

Comment:
supposing that one should run `crab_submit.py` in `$CMSSW_BASE/src`, the path to lumiMask should be `TauMLTools/Production/crab/configs/Run3L2/lumisection_EphemeralHLTPhysics_json.txt`